### PR TITLE
Format CMake files using gersemi

### PR DESCRIPTION
**Pull request by afuller-TT**
Introduced an auto-formatter for CMake files & ran it over the repo.

No functional change.

### CODE REVIEW

**Comment by tt-aho** [comment]
Will this be added to the post-commit static checks? Otherwise people can skip the pre-commit hook

**Comment by blozano-tt** [comment]
> Will this be added to the post-commit static checks? Otherwise people can skip the pre-commit hook

We plan to

**Review by blozano-tt — APPROVED** [review]
Beautiful

**Review by tt-aho — APPROVED** [review]

**Review by mo-tenstorrent — APPROVED** [review]

**Review by tt-rkim — APPROVED** [review]
Cool! why is the name gersemi?

**Review by aliuTT — APPROVED** [review]

**Review by yan-zaretskiy — APPROVED** [review]

File: cmake/fetch_boost.cmake [hunk 1]
```diff
@@ -10,17 +9,19 @@ function(fetch_boost_library BOOST_PROJECT_NAME)
...
-            ITEMS ${BOOST_INTERFACE_LINK_LIBRARIES})
-      if(NOT TARGET ${BOOST_INTERFACE_LINK_LIBRARY}
-         AND BOOST_INTERFACE_LINK_LIBRARY MATCHES "^Boost::([a-z0-9_]+)$")
-        fetch_boost_library(${CMAKE_MATCH_1})
-      endif()
-    endforeach()
-  endif()
+    if(NOT BOOST_INTERFACE_LINK_LIBRARIES STREQUAL BOOST_INTERFACE_LINK_LIBRARIES-NOTFOUND)
+        foreach(BOOST_INTERFACE_LINK_LIBRARY IN ITEMS ${BOOST_INTERFACE_LINK_LIBRARIES})
+            if(
```
Comment (ayerofieiev-tt) [inline cmake/fetch_boost.cmake:16]: Can this be improved?

**Review by ayerofieiev-tt — APPROVED** [review]
It is good to have a consistent format. Thank you

**Pull request merged**